src/Exception/Database/DatabaseExceptionTrait.php

```diff
@@ -7,7 +7,7 @@
     /** @var string */
     protected $driver;
 
-    private static $driverNames = [
+    protected static $driverNames = [
```

Comment:
Ugh … OK … we were both off on tangents :laughing: 

Short version: inheritance. So yes, this PR is actually correct. I am however going send it to `release/3.3` … as :bug: